content/docs/next-release/deployment.md

```diff
@@ -88,6 +88,9 @@ docker run \
 In the future we will support different service discovery systems to dynamically load balance
 across several collectors ([issue 213](https://github.com/jaegertracing/jaeger/issues/213)).
 
+Or add `--reporter.type=grpc` and `--reporter.grpc.host-port=jaeger-collector.jaeger-infra.svc:14250` to use gRPC
```

Comment:
This may be better above the previous `In the future` paragraph.

Do you need to say these options should be added to the agent?

Reply:
> Do you need to say these options should be added to the agent?

This is agent section so I think people will understand.